### DISCUSSION TITLE
Cómo traducir el verbo "bubbles"?

### DISCUSSION THREAD

**Discussion by Guirdo** [reactjs/es.reactjs.org#635] [Q&A] [answered]
Sucede en esta linea:

* onInvalid: An Event handler function. Fires if an input fails validation on form submit. Unlike the built-in invalid event, the React onInvalid event **bubbles**.

Podria traducirce como "el evento onInvalid aparece", o "emerge"? O quizás "burbujea"?

**Comment by manuelrodgzz** [comment, marked as answer]
Justo me dirigía a hacer esta pregunta ya que estoy traduciendo la documentación de `textarea`. [En este sitio](https://developer.mozilla.org/es/docs/Web/API/Event/bubbles) lo tradujeron como _"Se propaga"_. A mi parecer es más común encontrarse con el término _propagación_ en el mundo de la programación por lo que creo que se entiende mejor.

**Reply by Guirdo** [reply]
Tienes razón! No lo habia pensado. Gracias!

**Reply by carburo** [reply]
¡Gracias por la respuesta! Ya lo he añadido a la guía de traducción.
